src/sage/algebras/lie_algebras/lie_algebra_element.pyx

```diff
@@ -833,21 +835,34 @@ cdef class StructureCoefficientsElement(LieAlgebraMatrixWrapper):
             if v != zero:
                 yield (I[i], v)
 
-    cpdef to_vector(self, bint sparse=False):
+    cpdef _vector_(self, bint sparse=False, order=None):
         """
         Return ``self`` as a vector.
 
         EXAMPLES::
 
             sage: L.<x,y,z> = LieAlgebra(QQ, {('x','y'): {'z':1}})
-            sage: a = x + 3*y - z/2
-            sage: a.to_vector()
-            (1, 3, -1/2)
+            sage: a = x + 3*y - z/5
+            sage: vector(a)
+            (1, 3, -1/5)
         """
         if sparse:
             return self.value.sparse_vector()
         return self.value
 
+    cpdef to_vector(self, bint sparse=False, order=None):
```

Comment:
why can it just ignore the `order` parameter?
(Of course I see that the original code did the same.)

Reply:
Because the implementation is based on a fixed order and it is added for (signature) compatibility with the generic implementation.